app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { getCategories } = require("./controllers/app.controllers");
+
+const app = express();
+
+app.get("/api/categories", getCategories);
+
```

Comment:
You should add a route here for your 404 error. _hint hint: look at app.all('/*') and think about a new error handling function_.  (you'll need an error controller)